doc/specs/stdlib_io.md

```diff
@@ -223,3 +223,51 @@ program demo_savenpy
     call save_npy('example.npy', x)
 end program demo_savenpy
 ```
+
+## `getline`
```

Comment:
What do you think about `readline` for the name? It seems closer to the description and what the subroutine does. "get" is appropriate, but less specific than "read". `readline` is also how Python and Julia call it.

Reply:
I'm associating `readline` with the GNU readline library: https://en.wikipedia.org/wiki/GNU_Readline

Reply:
Do you think there's a risk that some users may confuse a Fortran subroutine `readline` with GNU readline?

GNU readline (and Python, Julia) aside, do you like `getline` better than `readline`?

I'd like to get some more feedback on this, let's discuss it on the monthly call.

Reply:
There is `std::getline` in C++ (https://www.cplusplus.com/reference/string/string/getline) and the C `getline` procedure (https://man7.org/linux/man-pages/man3/getline.3.html).

Reply:
There are several C interfaces to readline, and readline is well-known enough I have avoided using that name, also I make a habit of prefixing anything that is a C interface with system_* so it it clearer to me it is introducing a C dependency;  but I find I like having a low-level and a high-level read of a line  [(see getline and read_line])( https://urbanjost.github.io/M_io/man3.html) so wondering if  a  line_get, and line_read, and line_edit, ... family might emerge?  Might just be personal taste by I have had similiar functionality for a long time and I like a "read a line as-is" sometimes, and a "read a line and get rid of likely DOS line terminators and allow for line continuation with backslash" version, and a mini-editor (using  https://urbanjost/github/M_history of course :>) and when I am sure it will be available an interface to the C readline (which possibly introduces GNU issues but it the most supported one remaining that I know of) 

Since users can rename these (but not if they become true intrinsics!) maybe std_getline, std_readline, would be a convention to use?

Reply:
As agreed on the call, let's go forward with `getline`, as is.